packages/shared/src/components/auth/PreparingYourFeed.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import LogoIcon from '../../svg/LogoIcon';
+import LogoText from '../../svg/LogoText';
+import classed from '../../lib/classed';
+
+interface PreparingYourFeedProps {
+  isAnimating: boolean;
+}
+
+export const OnboardingContainer = classed(
+  'div',
+  'flex flex-col overflow-x-hidden items-center min-h-[100vh] w-full h-full max-h-[100vh] flex-1 z-max',
```

Comment:
Just wondering did we not fix these 100vh to be able to use tailwinds ones?

Reply:
I just copy-pasted this from the other file, but sure I will check it.